virtio-devices/src/block.rs

```diff
@@ -215,31 +218,42 @@ impl BlockEpollHandler {
             } else {
                 desc_chain
                     .memory()
-                    .write_obj(VIRTIO_BLK_S_OK as u8, request.status_addr)
+                    .write_obj(VIRTIO_BLK_S_OK, request.status_addr)
```

Comment:
This change should not be in this PR  - this change was recently made to fix an issue. Perhaps you could rebase and make sure this isn't included in this PR.

Reply:
I think that's something introduced by cargo fmt, already undo. May I know which issue is this change target to fix?